common/formatting/align.cc

```diff
@@ -117,6 +130,133 @@ std::ostream& operator<<(std::ostream& stream, const AlignmentCell& cell) {
   return stream;
 }
 
+template <typename ValueType, typename Cell>
```

Comment:
I think new non-trivial functions should have a short description of what they are doing ("Recursively ...") and what their parameters are (e.g. what is the input, what is expected from the get_cell_label callback.

Maybe we should also typedef or using the type for the get_cell_label to make things more readable, and make it easier to re-use (the ColumnsTreeFormatter below for instance also takes the same type of callback). It will also make the parameter-list a little more readable with a shorter name e.g. `const CellLabelGetterFun& get_cell_label`

Reply:
Fixed in [parent PR](https://github.com/chipsalliance/verible/pull/865)